### ISSUE TITLE
[1.11.2] Unresponsive server using ExtraPlanets

### ISSUE THREAD

**Issue by hugeblank** [MJRLegends/ExtraPlanets#149]
### ---Issue Report---
**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** Yes

### Description of Issue
The issue occurs on my server [Sonar X](https://kaos-network.com/servers/sonarx). After I updated to the latest version of EP, and started the server. Everything loaded fine, but at the point where the server should have become open to the players (it opened), the server became completely unresponsive, timed out any joining player, and wouldn't take console commands. _This isn't a crash_. There is no indication that the server goes down, it just sits there not responding to any message I give it.

### Steps to Reproduce Issue
1. Create a server using the mod list provided on Sonar X's webpage (for consistency)
2. Make sure you install the latest version of ExtraPlanets
3. Start the server and wait for the point where players can join. This is the step the issue will occur in.

**Version of Mod using:** 0.3.5

**Where did this bug happen:** Server

### Additional Information
None other than this is without a doubt extra planets. After removing the mod, the server booted, and responded perfectly fine.

**Comment by MJRLegends** [comment]
Please send me a full server log file

**Comment by hugeblank** [comment]
Closing issue as I found the cause. Mars was corrupted and rather than crashing, it just lagged the server to oblivion. After deleting the dimension, the problem went away. 

The reason I thought this had something to do with your mod was because I moved the original world folder, allowing the generation of a new one and tried without the mod. As is evident, the mod removal did nothing, but the moving of the world folder did the trick. 

My apologies for the false alarm...

Off topic: is there a place I can view the changelog of the mod? It would be really nice to have.
